src/m365/yammer/commands/yammer-search.ts

```diff
@@ -285,8 +290,48 @@ class YammerSearchCommand extends YammerCommand {
                 url: group.url
               }
             }));
+          } else if (show === "summary") {
+            logger.log(this.summary)
           } else { 
-            logger.log(this.summary);
+            let results: YammerBasicSearchResponse[] = [];
+            results = [...results, ...this.messages.map((msg) => {
```

Comment:
Maybe it's just my limited understanding of the code, but how can you cast the same values to different types? It seems confusing

Reply:
the different sets of objects contain the same properties as strings. This output works fine